docs/release/release_0_4_17.md

```diff
@@ -12,6 +12,38 @@ https://github.com/napari/napari
 
 ## Highlights
 
+### Minimum blending for a better light theme experience
```

Comment:
I'm not sure if there's a desired style for highlights, but I kind of like sub-sections. Can change to a list or something else if desired.